apps/browser/app/page.tsx

```diff
@@ -23,20 +23,23 @@ import { useRouter } from "next/navigation";
 import { useAtom } from "jotai";
 import { v4 as uuid } from "uuid";
 import { InMemoryFile } from "@nerfzael/memory-fs";
+import { useChats } from "@/lib/queries/useChats";
+import { ChatApi } from "@/lib/api/ChatApi";
```

Comment:
NIT: I've been trying to keep consistent ordering of imports, so would prefer these move next to their respective friends (i.e. `@/lib/api/...` and `@/lib/queries/...`